packages/compass-crud/src/components/document.tsx

```diff
@@ -32,6 +33,20 @@ const Document = (props: DocumentProps) => {
     return new HadronDocument(_doc as any);
   }, [_doc]);
 
+  const changeQuery = useChangeQueryBarQuery();
+
+  const onAddToQuery = useCallback(
```

Comment:
I've seen this in other React codebases and I believe we're also (sort of) using this in Compass.
```suggestion
  const handleAddToQuery = useCallback(
```